File: nysol_python/mcmd/methods/convTex/db/smp/mvcat.py
# ================================================
# コマンドマニュアル自動作成用サンプルコードデータ
# ================================================
db={}
db['name']='mvcat'

############################### IDATA
db['idatas']=[]

data={}
data['name']='dat1.csv'
data['text']='''
items1,items2,items3,items4
b a c,b,x,y
c c,,x,y
e a a,a a a,x,y
'''
db['idatas'].append(data)

############################### SCRIPTS
db['scripts']=[]

script={}
script['title']='ワイルドカードを利用した例'
script['comment']='''
'''
script['sh_code']='''
mvcat vf=items* a=items i=dat1.csv o=rsl1.csv
'''
script['py_code']='''
nm.mvcat(vf="items*", a="items", i="dat1.csv", o="rsl1.csv").run()
'''
script['odatas']='rsl1.csv'
db['scripts'].append(script)

File: olddoc/mcmd/jp/examples/mvcat.rb
#!/usr/bin/env ruby
# coding: utf-8
require "./mkTex.rb"

File.open("dat1.csv","w"){|fpw| fpw.write(
<<'EOF'
items1,items2,items3,items4
b a c,b,x,y
c c,,x,y
e a a,a a a,x,y
EOF
)}

############## 例1
title="ワイルドカードを利用した例"
comment=<<'EOF'
EOF
scp=<<'EOF'
more dat1.csv
mvcat vf=items* a=items i=dat1.csv o=rsl1.csv
more rsl1.csv
EOF
run(scp,title,comment)
